server/services/CardService.js

```diff
@@ -52,6 +54,10 @@ class CardService {
             logger.info(err);
         });
     }
+
+    getRestrictedList() {
+        return JSON.parse(fs.readFileSync(path.join(__dirname, '../../throneteki-json-data/restricted-list.json')));
```

Comment:
This has the potential to block the entire server if this read goes wrong for any reason, though I'm not sure what the alternative is.  Last I read the async file stuff wasn't that great.

Reply:
Originally I had this as a simple `require` at the top of the file. However, doing so forces you to restart the lobby server anytime data gets updated. Not a big deal in terms of the restricted list, but if we move to loading packs / cards directly from file we'll probably want hot-reload in general.

I can change this method to return a promise instead and use the normal async `readFile` instead if that will alleviate potential problems.